test/test.bats

```diff
@@ -107,6 +107,13 @@ EOF
     nix-build -A rosPackages.jazzy.ros-node
 }
 
+@test "--output-as-pkg-dir" {
+    ros2nix --output-as-ros-pkg-name $(find ws/src -name package.xml)
+    assert [ -f ros_node.nix ]
+    assert [ -f library.nix ]
```

Comment:
Those asserts will need to be changed too. They should probably test the existence of `ros-node/package.nix` and `library/package.nix`.